repos/fdbt-reference-data-service/src/retrievers/serverless.yml

```diff
@@ -1,199 +1,200 @@
 service: reference-data-service-retrievers
 
 provider:
-    name: aws
-    region: eu-west-2
-    runtime: python3.8
-    stage: ${opt:stage, 'test'}
-    stackName: ${self:service}-${self:provider.stage}
-    lambdaHashingVersion: 20201221
-    iam:
-        role:
-            statements:
-                - Effect: 'Allow'
-                  Action:
-                      - 'ssm:GetParameter'
-                  Resource: '*'
-                - Effect: 'Allow'
-                  Action:
-                      - 's3:PutObject'
-                      - 's3:GetObject'
-                  Resource:
-                      - !Sub arn:aws:s3:::fdbt-csv-ref-data-${self:provider.stage}/*
-                      - !Sub arn:aws:s3:::fdbt-txc-ref-data-${self:provider.stage}/*
-                      - !Sub arn:aws:s3:::fdbt-txc-ref-data-zipped-${self:provider.stage}/*
-                - Effect: 'Allow'
-                  Action:
-                      - 'cloudwatch:PutMetricData'
-                  Resource: '*'
-    s3:
-        txcZippedData:
-            name: fdbt-txc-ref-data-zipped-${self:provider.stage}
-            publicAccessBlockConfiguration:
-                BlockPublicAcls: true
-                BlockPublicPolicy: true
-                IgnorePublicAcls: true
-                RestrictPublicBuckets: true
-            bucketEncryption:
-                ServerSideEncryptionConfiguration:
-                    - ServerSideEncryptionByDefault:
-                          SSEAlgorithm: AES256
-            lifecycleConfiguration:
-                Rules:
-                    - Id: ExpiryRule
-                      Status: Enabled
-                      ExpirationInDays: '5'
+  name: aws
+  region: eu-west-2
+  runtime: python3.8
+  stage: ${opt:stage, 'test'}
+  stackName: ${self:service}-${self:provider.stage}
+  lambdaHashingVersion: 20201221
+  iam:
+    role:
+      statements:
+        - Effect: "Allow"
```

Comment:
Are the formatting changes intentional, or the result of an IDE not using the correct formatter?